q2_demux/_summarize/_visualizer.py

```diff
@@ -59,9 +59,11 @@ def _subsample_paired(fastq_map):
     for fwd, rev, index in fastq_map:
         file_pair = zip(_read_fastq_seqs(fwd), _read_fastq_seqs(rev))
         for i, (fseq, rseq) in enumerate(file_pair):
-            min_seq_len['forward'] = min(min_seq_len['forward'], len(fseq[1]))
-            min_seq_len['reverse'] = min(min_seq_len['reverse'], len(rseq[1]))
             if i == index[0]:
+                min_seq_len['forward'] = min(min_seq_len['forward'],
```

Comment:
`_subsample_single` will also need this update, btw.

Reply:
Good catch! :baseball: